pkg/policy/node.go

```diff
@@ -26,11 +26,16 @@ import (
 
 // Node to define hierarchy of rules
 type Node struct {
-	path      string
-	Name      string           `json:"name"`
-	Parent    *Node            `json:"-"`
-	Rules     []PolicyRule     `json:"rules,omitempty"`
-	Children  map[string]*Node `json:"children,omitempty"`
+	path     string
+	Name     string           `json:"name"`
+	Parent   *Node            `json:"-"`
+	Rules    []PolicyRule     `json:"rules,omitempty"`
+	Children map[string]*Node `json:"children,omitempty"`
+
+	// CoverAll marks policy nodes which should always have their
+	// rules applied regardless of whether the name/path of the node
+	// matches any of the labels in the search context.
+	CoverAll  bool `json:"coverAll,omitempty"`
```

Comment:
The mean of `coverAll` is that the name of the policy node should not be a requirement for any of the rules of the node to be considered. The coverage of the rules is defined by the coverage of the rules themselves. The node itself is a pure "packaging" construct. This allows to map k8s NetworkPolicy to policy nodes without requiring the name of the policy node to somehow correlate to the coverage of the actual rules.

I think you are saying the same thing but wanted to make sure.